news/2024/2024-01-26-mwc-7k-ro24-summary.md

```diff
@@ -0,0 +1,83 @@
+
+![](/wiki/Tournaments/MWC/2024_7K/img/mwc7k2024-banner.jpg)
+
+Continuing on from the osu! World Cup [recaps](https://osu.ppy.sh/home/news/2023-11-28-osu-world-cup-2023-finals-recap), you can once more expect a summary of the latest and greatest of MWC, made possible by community members just like you! This year, **for the first time ever**, the tournament kicked off with a Round of 24! Due to increased interest from various countries that have both applied and qualified, we have been blessed with an extra round and exciting matches straight away!
+
+Before getting into the highlight matches, let's quickly reflect (or familiarise) ourselves with the Round of 24 mappool.
+
+### Mappool showcase
+
+[**View the VOD here!**](https://www.twitch.tv/videos/2032775000)
+
+[**Download the pool here! (50 MB)**](https://packs.ppy.sh/P233%20-%20osu!mania%207K%20World%20Cup%202024%3A%20Round%20of%2024.zip)
+
+With a new tournament comes a suite of new custom beatmaps for everyone to enjoy! This time around, the mapping team is making active efforts to ensure some of these picks are available to be Ranked after the tournament ends. Stay tuned for all [osu! originals](/wiki/osu!_originals) that are up to come — you may see them Ranked earlier than you'd think so!
+
+### Match highlights
+
+We may just be getting started, but we already had a few matches that are worth highlighting take place.
+
+### ::{ flag=BR }:: Brazil vs. ::{ flag=PT }:: Portugal
+
+[**View the VOD here!**](https://www.twitch.tv/videos/2038389122)
+
+Brazil vs. Portugal was a match that went as one would have expected on paper, but there is one thing during this match that we would like to highlight.
+
+Almost every single 7K tournament player in the recent years have that one annoying skill set to deal with. Usually, early-stage LN coordination charts (The "Oh boy, 2* in the pool!" trap) faces the fate of being the most banned chart in a mappool. In this match, however, the stars aligned in such a way that LN coordination was left as an open option.
+
+Portugal was down 4—0 coming into their third pick of the match, with only a limited amount of options left to turn the match around. ::{ flag=PT }:: [Rodrig0v](https://osu.ppy.sh/users/138866), their team captain, made the decision to exhaust the LN pool and selected the notorious LN1 chart, [Mitsukiyo - Theme_37](https://osu.ppy.sh/beatmapsets/2119592#mania/4452483). Clocking in at 69 BPM, and with only 313 long notes, absolutely no one can afford any misses. The chart started and everyone's combos and accuracies started dropping like flies except for one person holding on (literally!). It was Rodrig0v who is doing more than enough to give Portugal that sweet, sweet advantage. While Brazil will eventually have an S-rank score coming from ::{ flag= BR }:: [LothusFangirl](https://osu.ppy.sh/users/4685756), it wasn't enough to cover for the heroics made by Portugal's captain.
```

Comment:
what exactly is the map notorious for?

Reply:
Lower end LN coordination has always been deceptively hard. It received the most amount of bans, [achieved the lowest average score and accuracy by a landslide](https://docs.google.com/spreadsheets/d/1_xrYtyZ68mQpsfc7q9CVetV9E9BYxyj7Wg28w1d3esY/edit#gid=1554219608), and all of this while being just a 2.07* beatmap.